### DISCUSSION TITLE
Entities Buttons row overflow to a new row on narrow screens

### DISCUSSION THREAD

**Discussion by anthonyma94** [home-assistant/frontend#10959] [Dashboard]
Ever since the update to the entities buttons row UI (2021.12), the extra padding and borders take up a lot of space on narrow screens. By default, the buttons are organized in a single row no matter the screen width, making the buttons hidden behind a horizontal scroll bar. I believe it would be better if the buttons can inherit flexbox-like capabilities, where it overflows to a second row if needed.

**Comment by jlpouffier** [comment]
Yes.
Or at least give us a choice for that.

Our dashboards are created to allow critical action available with a single tap of a button...
The horizontal scrolling is a weird default choice. I would love to have a choice.

**Comment by spacegaier** [comment]
> By default, the buttons are organized in a single row no matter the screen width...

Not completely true, on big screens (= desktop) the buttons actually break into multiple rows, so probably a CSS topic that only affects narrow screens.
